### ISSUE TITLE
Vue Dev Tools (Browser Extension) in Browser Inspector disappear when Nuxt Devtools are enabled

### ISSUE THREAD

**Issue by cheike569** [nuxt/nuxt#26461]
### Environment

- Operating System: `Darwin`
- Node Version:     `v18.19.1`
- Nuxt Version:     `3.11.1`
- CLI Version:      `3.11.1`
- Nitro Version:    `2.9.4`
- Package Manager:  `yarn@1.22.21`
- Builder:          `-`
- User Config:      `devtools`, `vite`, `app`, `modules`, `imports`, `runtimeConfig`, `devServer`, `ssr`, `nitro`, `tailwindcss`, `i18n`, `primevue`
- Runtime Modules:  `@nuxtjs/tailwindcss@6.11.4`, `nuxt-primevue@0.3.1`, `@nuxtjs/i18n@8.2.0`, `@pinia/nuxt@0.5.1`
- Build Modules:    `-`

### Reproduction
```
// nuxt.config.ts
export default defineNuxtConfig({
    devtools: {enabled: true},
});
```

### Describe the bug

With Nuxt 3 Devtools enabled, the original Vue Devtols (Chrome Browser Extension) disappears from the Inspector.

```
// nuxt.config.ts
export default defineNuxtConfig({
    devtools: {enabled: true},
});
```

<img width="338" alt="Screenshot 2024-03-23 at 2 48 39 PM" src="https://github.com/nuxt/nuxt/assets/244036/345a7c2e-3b0f-4a45-b1de-a63f72962f4a">

**Disabling the Nuxt Devtools will bring back the Vue Devtools:**

```
// nuxt.config.ts
export default defineNuxtConfig({
    devtools: {enabled: false},
});
```
<img width="912" alt="Screenshot 2024-03-23 at 2 49 14 PM" src="https://github.com/nuxt/nuxt/assets/244036/be76eace-4689-4cda-a285-faf4bf361cd9">

**Expected behavior: Both Dev Tools should be visible when Nuxt Dev Tools are turned on.**

**Comment by JhumanJ** [comment]
Thank you! I finally got vue dev tools back after setting `devtools` to  `{enabled: false}`!

**Comment by NtchPlayer** [comment]
Yup, I have the same problem and disabled the Nuxt Devtools works for me too ! thx 😄

**Comment by danielroe** [comment]
This is an issue in https://github.com/nuxt/devtools. Keeping it open here so other people know.

A solution for now it to explicitly specify a `@nuxt/devtools` version in your `package.json` of `1.0.8` - you do not need to disable devtools entirely.

**Comment by JhumanJ** [comment]
I might have done it wrong, but keeping it at 1.0.8 wasn't enough for me. Thanks anyway for the update!
https://github.com/nuxt/devtools/issues/634#issuecomment-2016795748

**Comment by antfu** [comment]
Should be fixed with `@nuxt/devtools@1.1.4`
